### PR TITLE
You can now throw mobs across lava without them burning

### PR DESCRIPTION
**Pull request by silicons**
????? physics???

### CODE REVIEW

File: code/game/turfs/simulated/lava.dm [hunk 1]
```diff
@@ -89,6 +89,8 @@
 	if (AM)
 		thing_to_check = list(AM)
 	for(var/thing in thing_to_check)
+		if(thing.throwing)
+			continue	//Passing over it
 		if(isobj(thing))
 			var/obj/O = thing
 			if((O.resistance_flags & (LAVA_PROOF|INDESTRUCTIBLE)) || O.throwing)
```
Comment (MrStonedOne) [inline code/game/turfs/simulated/lava.dm]: but... it already checks this here.

Comment (AnturK) [inline code/game/turfs/simulated/lava.dm]: This is probably i ded balance change in disguise.

Comment (GunHog) [inline code/game/turfs/simulated/lava.dm]: Yes, but here it only prevents flying _objects_ burning in lava, not mobs.

**Comment by Jordie0608** [comment]
Because who needs thermodynamic convection right?

**Comment by kevinz000** [comment]
no this isnt i ded the last time i ded to it was half a year ago i die to chasms more than lava who even dies to something that bright and noticable

**Comment by FantasticFwoosh** [comment]
There's a problem with this in that 2 gulag prisoners can just throw the other across the lava-gap

**Comment by InsaneHyena** [comment]
Convection, schmovection...

**Comment by Repukan** [comment]
Do we really need this?

**Comment by Repukan** [comment]
Did they gulag you and Marisa and you got burnt up or something?

**Comment by MMMiracles** [comment]
This seems like a good way to make the gulag even more of a joke than it already is. 

On the plus side this also buffs jump boots to allow river crossing.

**Comment by ma44** [comment]
inb4 syndie base

**Comment by PraiseRatvar** [comment]
@Repukan 
...

**Comment by Ergovisavi** [comment]
>Object's entire purpose is to act as a damaging roadblock requiring you to either route around it or secure a hard to acquire/costly method of traversing it. We should add a piss easy way of getting around it!

Stop that.

**Comment by kevinz000** [comment]
honestly i didn't think about the gulag before I did this 🤷‍♂️

**Pull request closed without merging**